ui/src/components/sermon-notes/SermonNotesContainer.js

```diff
@@ -21,6 +21,7 @@ const SermonNotesContainer = (props) => {
   const [userSermonNotes, setUserSermonNotes] = useState();
   const [editUserSermonNotes, setEditUserSermonNotes] = useState();
   const toast = useToast();
+  const [isLogin, setLogin] = useState(false);
```

Comment:
Since u change the way the button is rendered, i think this is unused? Can check n delete along w the useEffect if its unused.